src/Api/API.php

```diff
@@ -703,40 +703,40 @@ protected function getItem($itemtype, $id, $params = [])
                         'glpi_softwareversions.id AS softwareversions_id',
                         'glpi_items_softwareversions.is_dynamic',
                         'glpi_softwareversions.states_id',
-                        'glpi_softwares.is_valid'
+                        'glpi_softwares.is_valid',
                     ],
                     'FROM'      => 'glpi_items_softwareversions',
                     'LEFT JOIN' => [
                         'glpi_softwareversions' => [
                             'ON' => [
                                 'glpi_items_softwareversions' => 'softwareversions_id',
-                                'glpi_softwareversions'       => 'id'
-                            ]
+                                'glpi_softwareversions'       => 'id',
+                            ],
                         ],
                         'glpi_softwares'        => [
                             'ON' => [
                                 'glpi_softwareversions' => 'softwares_id',
-                                'glpi_softwares'        => 'id'
-                            ]
-                        ]
+                                'glpi_softwares'        => 'id',
+                            ],
+                        ],
                     ],
                     'WHERE'     => [
                         'glpi_items_softwareversions.items_id'   => $id,
                         'glpi_items_softwareversions.itemtype'   => $itemtype,
-                        'glpi_items_softwareversions.is_deleted' => 0
+                        'glpi_items_softwareversions.is_deleted' => 0,
                     ],
                     'ORDERBY'   => [
                         'glpi_softwares.name',
-                        'glpi_softwareversions.name'
-                    ]
+                        'glpi_softwareversions.name',
```

Comment:
Please avoid those kind of changes, this is useless and makes PRs difficult to read

Reply:
The changes have been removed [7f40cea](https://github.com/glpi-project/glpi/pull/17364/commits/7f40ceaa44c66b0a7e09bd4a2cd5a7b97723027a)